#! /bin/sh /usr/share/dpatch/dpatch-run
## 0020-upstream-client-literal-prefix.dpatch by Sven Mueller <sven@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Upstream patch: Prefix client responses by C:

@DPATCH@
--- a/imtest/imtest.c
+++ b/imtest/imtest.c
@@ -1,7 +1,7 @@
 /* imtest.c -- IMAP/POP3/NNTP/LMTP/SMTP/MUPDATE/MANAGESIEVE test client
  * Ken Murchison (multi-protocol implementation)
  * Tim Martin (SASL implementation)
- * $Id: imtest.c,v 1.108 2006/05/25 15:23:07 murch Exp $
+ * $Id: imtest.c,v 1.109 2006/08/30 16:58:49 murch Exp $
  *
  * Copyright (c) 1998-2003 Carnegie Mellon University.  All rights reserved.
  *
@@ -1083,7 +1083,8 @@ int auth_sasl(struct sasl_cmd_t *sasl_cm
 
 	    /* send to server */
 	    if (sendliteral) {
-		printf("{%d+}\r\n", inbase64len);
+		printf("%s{%d+}\r\n",
+		       initial_response ? "" : "C: ", inbase64len);
 		prot_printf(pout, "{%d+}\r\n", inbase64len);
 		prot_flush(pout);
 	    }
